features/accessibility.feature

```diff
@@ -0,0 +1,39 @@
+@javascript
+Feature: Accessibility of pages
+  As a customer I want Pension Wise pages to be fully accessible
+  so I can navigate the website regardless of my disability
+
+  Scenario: Homepage of the site is accessible
+    When I visit the homepage
+    Then the page should be accessible
+
+  Scenario Outline: Typical pages of the site are accessible
+    When I visit <Slug> page
+    Then the page should be accessible
+
+    Examples:
+      | Slug                     |
+      | facebook-landing         |
+      | leave-pot-untouched      |
+      | tax                      |
+      | summary-document/new     |
+      | feedback/new             |
+
+  @vcr
+  Scenario: Location page is accessible
+    Given I have searched for appointment locations near to a valid postcode
+    When I drill down into a specific search result
+    Then the page should be accessible
+
+  Scenario: Appointment summary page is not accessible until errors have been fixed
+    When I visit the digital appointment summary generator
+    And I select 55 or over as my age range
+    And I view the appointment summary
+    Then the page should be accessible
+
+  @booking_locations
+  Scenario: Slot picker for face to face booking is not accessible while we develop our own slot picker
+    Given a location is enabled for online booking
+    When I browse for the location
+    And I opt to book online
+    Then the page should not be accessible
```

Comment:
Was there another way of wording this that better described our situation? I think it was just an alias for this one.